extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -446,13 +447,11 @@
 						}
 					}
 
-					//readd events
+					//re-add events
 					ArticleComments.addHover();
-					//clear error box
-					$('#article-comm-info').html('');
```

Comment:
This article-comm-info element has `display: none` by default, and shows up, unstyled, at the top of the page.  It doesn't seem like it was ever finished.  Using BannerNotifications instead.